tests/org.codechecker.eclipse.rcp.it.tests/resources/cproject/src/cproject.c

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Can you create a separate PR adding this new test? If yes please split it up.
Adding a test later will be easier if it is visible if we see what are the changes required to add a test, right now it is mixed with the restructure changes.

Reply:
The PR for the separate test can be found here #154